catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/models.properties

```diff
@@ -83,6 +83,8 @@ jsonpath
 jtaTransactionErrorHandler
 kamelet
 kubernetesServiceDiscovery
+langChain4j
+langchain4j
```

Comment:
Is this right?

Reply:
Hm, that looks suspicious. Let me take a look.

Reply:
I had named it differently in two occasions and there were leftovers. It's fixed now, thanks!